pennylane/qnode.py

```diff
@@ -679,7 +691,7 @@ def check_op(op):
         for op in self.ops:
             check_op(op)
 
-        ret = self.device.execute(self.queue, self.ev, self.variable_ops)
+        ret = self.device.execute(self.circuit.operations, self.circuit.observables, self.variable_ops)
```

Comment:
I really like this naming, it makes it really easy to read; 'circuit operations', 'circuit observables'.